ccg_nlpy/server/abstract_model.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+from ccg_nlpy.core.text_annotation import TextAnnotation
+
+
+class AbstractModel:
```

Comment:
Just keep things compatible with the Java side, how about we name it sth like `AbstractView`?

Reply:
What do we call a model that works inside a server? I can name this `Annotator`

Reply:
Makes sense.